src/main/kotlin/com/terraformation/backend/search/SearchFilter.kt

```diff
@@ -126,7 +127,7 @@ data class FieldNode(
   }
 
   override fun toExactSearch(): FieldNode {
-    return if (type == SearchFilterType.Fuzzy && values.any { it != null }) {
+    return if (type == SearchFilterType.ExactOrFuzzy && values.any { it != null }) {
```

Comment:
If I understand correctly, now the `toExactSearch` conversion will only happen for `ExactOrFuzzy` nodes, whereas for `Fuzzy` node `this` is returned, meaning it will stay fuzzy?

Reply:
Yes, that's the idea. We're removing the "try an exact search first" behavior when the filter type is `Fuzzy`.